.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
+on:
+  push:
+    branches: [main]
+    paths: [src/**, action.yml, .github/wiki/**, .github/workflows/test.yml]
+  pull_request:
+    branches: [main]
+    paths: [src/**, action.yml, .github/wiki/**, .github/workflows/test.yml]
```

Comment:
Out of paranoia, I changed this repo's settings to require approval for workflow runs for *all* outside contributors. Deploy-on-PR can get pretty dangerous, otherwise :laughing: